### PR TITLE
Add Cursed Game Hunter achievement

### PR DESCRIPTION
**Pull request by dgodfrey**
Adds nodes for the Cursed Game Hunter achievement from BfA

### CODE REVIEW

File: plugins/08_BattleForAzeroth/zones/drustvar.lua [hunk 1]
```diff
@@ -845,3 +845,68 @@ map.nodes[21602880] = Safari.ShackCrab({
...
+--------------------------- CURSED GAME HUNTER --------------------------------
+-------------------------------------------------------------------------------
+
+local Cursed = Class('Cursed', Collectible, {
+    icon = 1027879,
+    group = ns.groups.CURSED_HUNTER,
+    sublabel = L['cursed_hunter_note']
+})
+
+function Cursed.getters:label() return "Cursed animals" end
```
Comment (Dathwada) [inline plugins/08_BattleForAzeroth/zones/drustvar.lua]: `Cursed.getters:label()` could be replaced by adding `label = L['Cursed animals']` to the Class.
And `L['Cursed animals']` should be added to localizations.

File: plugins/08_BattleForAzeroth/localization/enUS.lua [hunk 2]
```diff
@@ -38,6 +38,10 @@ L['dilbert_mcclint_note'] = 'Hey there, name\'s {npc:140461}, Infestation Manage
 L['fizzie_spark_note'] = 'You think your pets have a chance against my Azerite infused team? You wish!'
 L['michael_skarn_note'] = 'Just remember as we start this battle, you asked for this.'
 
+L['cursed_hunter_note'] = 'Kill one of every type of cursed animal to earn the achievement.'
+L['options_icons_cursed_hunter_desc'] = 'Display animal locations for the {achievement:13094} achievement.'
+L['options_icons_cursed_hunter'] = '{achievement:13094}'
```
Comment (Dathwada) [inline plugins/08_BattleForAzeroth/localization/enUS.lua:44]: I would like to request that you also add these lines to all the other localizations. But with the change that these are nil.
e.g. `L['cursed_hunter_note'] = nil` 
more infos: https://github.com/zarillion/handynotes-plugins/pull/195#issuecomment-1272416272

**Comment by Dathwada** [comment]
Hi @dgodfrey, thanks for the contribution. I took a quick look at this and would like to request some changes before we run the tests.

**Comment by dgodfrey** [comment]
Hopefully I didn't miss something.

**Comment by Krovikan** [comment]
Hello:

Drustvar.lua have some format error.

Regards

**Comment by Dathwada** [comment]
What do you think about adding the npc names?

```lua
map.nodes[53872072] = Cursed({
    label = '{npc:143929}',
    rewards = {Achievement({id = 13094, criteria = 1})} -- 69109})}
}) -- Cursed Game Hunter: Goats
```

**Comment by dgodfrey** [comment]
I've removed those extraneous Ids - can't remember what they were tbh.

**Comment by Ioney** [comment]
> I've removed those extraneous Ids - can't remember what they were tbh.

Its like a placeholder.
`{npc:143929}` will be replaced with the npc name in the current game language. For enUS it will be `Hexed Craghopper`.

**Comment by dgodfrey** [comment]
> > I've removed those extraneous Ids - can't remember what they were tbh.
> 
> Its like a placeholder. `{npc:143929}` will be replaced with the npc name in the current game language. For enUS it will be `Hexed Craghopper`.

I was referring to the "-- <number>" on the end of the rewards line. In the above case it was "-- 69109})}".... I think it might have been from when I was trying to figure out what the criteria Id were for the various bits of the achievement. :)

With reference to the npc's - I think there's sometimes 2 or 3 that can be killed for the criteria, hence I just put "Cursed Animals" as the label. I considered adding it as a sub-label though.

**Pull request merged**